README.md

```diff
@@ -1,118 +1,80 @@
-# Jenkins Folder Properties Plugin
+A Jenkins plugin which allows users with config permission to define
+properties for a folder which can then be used by any jobs contained
+within it or any of its sub-folders. 
 
-A Jenkins plugin which allows users with config permission to define properties for a folder which can then be used by
-any jobs contained within it or any of its sub-folders.
+You can find [this plugin's source code on
+Github](https://github.com/jenkinsci/folder-properties-plugin).
```

Comment:
I'd delete this paragraph since there is already a GitHub link provided from the plugins.jenkins.io page.
```suggestion
```